apps/civicsignalblog/src/payload/collections/Users.js

```diff
@@ -53,6 +56,104 @@ const Users = {
       },
       options: ROLE_OPTIONS,
     },
+    {
+      name: "defaultManagedApplication",
+      type: "select",
+      hasMany: false,
+      admin: {
+        isClearable: true,
+        isSortable: true,
+      },
+      options: applications,
```

Comment:
Should default to `research` app as that's what is already there.

Reply:
Makes sense I have made the change